src/app.js

```diff
@@ -69,12 +69,16 @@ if (process.env.KEEP_ALIVE) {
 	});
 }
 
+if (process.env.BODYPRASER_JSON_LIMIT === undefined) {
+	logger.warning('please set the environment variable BODYPRASER_JSON_LIMIT to 12mb for helpdesk to work correctly! (Default: 100kb)');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line 73 exceeds the maximum line length of 120. (max-len)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4570602)